RoslyJump.sln

```diff
@@ -16,41 +16,130 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{BFB7674D-6B82-4606-835C-D58A68B3C233}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
-		License.txt = License.txt
 		readme.md = readme.md
 	EndProjectSection
 EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "RoslyJump.Shared", "RoslyJump.Shared\RoslyJump.Shared.shproj", "{235831B2-F9EF-490F-B33D-ECFD1A137132}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RoslyJump.VSIX.2022", "RoslyJump.VSIX.2022\RoslyJump.VSIX.2022.csproj", "{CC52AD7A-C574-4530-A78B-3D02078918D7}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Scripts", "Scripts", "{287DC108-F0EB-497E-8D63-D603DCE2F070}"
+	ProjectSection(SolutionItems) = preProject
+		pre-build-phase-1.sh = pre-build-phase-1.sh
+		pre-build-phase-2.sh = pre-build-phase-2.sh
+		pre-release.sh = pre-release.sh
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "VSIX Shared Files", "VSIX Shared Files", "{E7B5721B-1F39-4367-A5D8-F4BAE914A197}"
+	ProjectSection(SolutionItems) = preProject
+		Shared\License.txt = Shared\License.txt
+		publishManifest.json = publishManifest.json
```

Comment:
Make sure both reside in the same location (root folder or shared folder, not both). After updating, don't forget to update the release pipeline. 

Currently:

```txt
		publishManifest.json = publishManifest.json
		Shared\publishManifest2022.json = Shared\publishManifest2022.json
```

Reply:
Moved to `\PublishManifests`